File: cpp/42e_1369A_FashionableLee.cpp
#include<bits/stdc++.h>

int main(){
    int t, n;
    std::vector<std::string> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        if (n % 4 == 0){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/42e_1369A_FashionableLee.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        if n % 4 == 0:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()